src/measure_theory/measure_space.lean

```diff
@@ -1049,7 +1233,13 @@ class probability_measure (μ : measure α) : Prop := (measure_univ : μ univ =
 /-- A measure `μ` is called finite if `μ univ < ⊤`. -/
 class finite_measure (μ : measure α) : Prop := (measure_univ_lt_top : μ univ < ⊤)
 
-export probability_measure (measure_univ)
+/-- Measure `μ` *has no atoms* if the measure of each singleton is zero.-/
+class no_atoms_measure (μ : measure α) : Prop :=
```

Comment:
I've also seen the term `continuous measure` for this property. The doc-string should mention this term.

Reply:
Renamed. Feel free to add "continuous measure" to the doc string. I don't want to do it myself because I never saw it (only "absolutely continuous" but this is a different property). I also commented on the definition. I don't want to change it in this PR but probably we should do it in the future.